template.yaml

```diff
@@ -14,11 +19,20 @@ Resources:
       Handler: app.lambda_handler
       Runtime: python3.6
       Role: !GetAtt BudgetMakerFunctionRole.Arn
+      Environment:
+        Variables:
+          NOTIFICATION_TOPIC_ARN: !Ref BudgetMakerNotificationTopic
+          SYNAPSE_TEAM_MEMBER_LIST_ENDPOINT: !Ref SynapseTeamMemberListEndpoint
+          AWS_ACCOUNT_ID: !Ref 'AWS::AccountId'
       Events:
-        FiveMinute: # Trigger every five minutes
+        OneMinute: # Trigger every minute
```

Comment:
Using more rapid schedule during development, will revert to longer intervals later.